docs/Game/Unreal/getting-started/02-sending-transactions.md

```diff
@@ -0,0 +1,346 @@
+---
+title: 02 Sending Transactions & Retrieving Data
+id: unreal-send-retrieve
+---
+
+This page lists the different methods and functions you can make
```

Comment:
Aren't functions and methods the same in this context? 

A full stop missing at the end of the sentence?